manim/utils/color.py

```diff
@@ -104,8 +104,9 @@ class Colors(Enum):
 
     .. code-block:: python
 
-       import manim.utils.color as C
-       C.WHITE   # -> '#FFFFFF'
+        >>> import manim.utils.color as C
+        >>> C.WHITE   
```

Comment:
Sorry, just a tiny issue: the trailing whitespace should be removed.

```suggestion
        >>> C.WHITE
```

Reply:
ok :D